service/storage_service.cc

```diff
@@ -777,10 +784,15 @@ class topology_coordinator {
         co_return std::make_pair(co_await retake_node(id), res);
     };
 
-    future<> handle_ring_transition(node_to_work_on&& node) {
+    // Returns `true` iff there was work to do.
+    future<bool> transition(group0_guard guard) {
```

Comment:
`handle_topology_transition()` if we are into descriptive names?

Reply:
ok